Kconfig.dependencies

```diff
@@ -127,6 +180,18 @@ config PSA_WANT_ALG_CBC_PKCS7
 config PSA_WANT_ALG_HKDF
 	default n
 
+config PSA_USE_CC3XX_KEY_AGREEMENT_DRIVER
+	default n if SOC_NRF52840 || SOC_NRF5340_CPUAPP
```

Comment:
I'm wondering if we could find a more generic reason for the if statement than specific SOC,
I thought that both nrf52 and nrf53 use crypto cell, so maybe this should be unconditionally default to false ? (as nrf54 do should not have and use this config? )

Reply:
the nrf crypto configuration is a minefield, I would rather not change this